### DISCUSSION TITLE
Is there controller support?

### DISCUSSION THREAD

**Discussion by bruhific** [skyline-emu/skyline#391] [Q&A] [answered]
Does it support generic controllers?

**Comment by iniMerchant64** [comment]
Yes it does. You can map inputs however you want in settings

**Comment by vicvenpet** [comment]
But if you're looking to use it with a keyboard it's being worked on by @MCredstoner2004

**Comment by PixelyIon** [comment, marked as answer]
Skyline does indeed support all kind of controllers as long as Android can pick them up.
